.github/workflows/ci.yml

```diff
@@ -144,7 +147,7 @@ jobs:
       - name: Install proto-related tools for Go
         run: |
           go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
-          go install github.com/containerd/protobuild@v0.2.0
+          go install github.com/containerd/protobuild@14832ccc41429f5c4f81028e5af08aa233a219cf
```

Comment:
Can we create a new release of protobuild?

Reply:
Sure

Reply:
I'd like to include https://github.com/containerd/protobuild/pull/56 in 0.3.0.

Reply:
Done.